File: Loadshop.DomainServices/Loadshop/Services/Data/LoadCarrierGroupData.cs
using System;

namespace Loadshop.DomainServices.Loadshop.Services.Data
{
    public class LoadCarrierGroupData
    {
        public long LoadCarrierGroupId { get; set; }
        public string GroupName { get; set; }
        public string GroupDescription { get; set; }
        public Guid CustomerId { get; set; }
        public string OriginAddress1 { get; set; }
        public string OriginCity { get; set; }
        public string OriginState { get; set; }
        public string OriginPostalCode { get; set; }
        public string OriginCountry { get; set; }
        public string DestinationAddress1 { get; set; }
        public string DestinationCity { get; set; }
        public string DestinationState { get; set; }
        public string DestinationPostalCode { get; set; }
        public string DestinationCountry { get; set; }
        public int LoadCarrierGroupTypeId { get; set; }
        public string LoadCarrierGroupTypeName { get; set; }
        //public CustomerData Customer { get; set; }
        public int CarrierCount { get; set; }
    }
}

File: Loadshop.Web.UI/src/app/shared/models/load-carrier-group-data.ts
import { LoadCarrierGroupCarrierData } from '.';
import { LoadCarrierGroupEquipmentData } from './load-carrier-group-equipment-data';

export interface LoadCarrierGroupData {
  loadCarrierGroupId: number;
  groupName: string;
  groupDescription: string;
  customerId: string;
  originAddress1: string;
  originCity: string;
  originState: string;
  originPostalCode: string;
  originCountry: string;
  destinationAddress1: string;
  destinationCity: string;
  destinationState: string;
  destinationPostalCode: string;
  destinationCountry: string;
  loadCarrierGroupTypeId: number;
  loadCarrierGroupTypeName: string;

  loadCarrierGroupEquipment: LoadCarrierGroupEquipmentData[];
}

export interface LoadCarrierGroupDetailData extends LoadCarrierGroupData {
  // TODO: customer: Customer;
  carrierCount: number;
  carriers: LoadCarrierGroupCarrierData[];
}

export const defaultLoadCarrierGroup: LoadCarrierGroupDetailData = {
  loadCarrierGroupId: null,
  groupName: null,
  groupDescription: null,
  customerId: null,
  originAddress1: null,
  originCity: null,
  originState: null,
  originPostalCode: null,
  originCountry: null,
  destinationAddress1: null,
  destinationCity: null,
  destinationState: null,
  destinationPostalCode: null,
  destinationCountry: null,
  loadCarrierGroupEquipment: [],
  loadCarrierGroupTypeId: null,
  loadCarrierGroupTypeName: null,

  // TODO: customer: Customer;
  carrierCount: null,
  carriers: [],
};
